mifospay/src/main/AndroidManifest.xml

```diff
@@ -71,8 +70,8 @@
         </activity>
         <activity
             android:name=".invoice.ui.InvoiceActivity"
-            android:theme="@style/AppTheme"
-            android:launchMode="singleTask">
+            android:launchMode="singleTask"
+            android:theme="@style/AppTheme">
```

Comment:
was this rearranging of lines necessary?

Reply:
Okay, I will.

Reply:
done